app/src/icons.md

```diff
@@ -0,0 +1,68 @@
+To install cd app and run npm install remixicon --save
```

Comment:
This isn't necessary, if it's installed to package-lock.json (which it should be) other developers need only run 'npm install'

Reply:
Oh cool! Thank You :)